src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -585,14 +590,28 @@ private function createEncoder($config, ContainerBuilder $container)
             ];
         }
 
+        if ('native' === $config['algorithm']) {
+            return [
+                'class' => NativePasswordEncoder::class,
+                'arguments' => [
+                    $config['time_cost'],
+                    (($config['memory_cost'] ?? 0) << 10) ?: null,
```

Comment:
memory_cost (from passwork_hash) is in `kib` but SodiumPasswordEncoder and NativePasswordEncoder accept bytes - this makes the conversion